docker/entrypoint.sh

```diff
@@ -0,0 +1,193 @@
+else
+  SQLITE_CONNECTION_STRING_API="Data Source=$BWP_DB_FILE_API;"
+  SQLITE_CONNECTION_STRING_ADMIN="Data Source=$BWP_DB_FILE_ADMIN;"
+  export ConnectionStrings__sqlite__api=${ConnectionStrings__sqlite__api:-$SQLITE_CONNECTION_STRING_API}
+  export ConnectionStrings__sqlite__admin=${ConnectionStrings__sqlite__admin:-$SQLITE_CONNECTION_STRING_ADMIN}
+fi
+
+#################
+# E-mail / SMTP #
+#################
+if [ -z "$BWP_SMTP_FROM" ] || [ "$BWP_SMTP_HOST" == "null" ]; then
+  echo "[Configuration] SMTP E-mail configuration not set. Writing to a local file instead in '/app/Admin/mail.md' or '/app/Api/mail.md'. See 'https://docs.passwordless.dev/guide/self-hosting/configuration.html'.";
+else
+  if [ -n "$BWP_SMTP_FROM" ] && [ "$BWP_SMTP_FROM" != "null" ]; then
+    export Mail__Smtp__From=$BWP_SMTP_FROM
+  fi
+  
+  if [ -n "$BWP_SMTP_USERNAME" ] && [ "$BWP_SMTP_USERNAME" != "null" ]; then
+    export Mail__Smtp__Username=$BWP_SMTP_USERNAME
+  fi
+  
+  if [ -n "$BWP_SMTP_PASSWORD" ] && [ "$BWP_SMTP_PASSWORD" != "null" ]; then
+    export Mail__Smtp__Password=$BWP_SMTP_PASSWORD
+  fi
+  
+  if [ -n "$BWP_SMTP_HOST" ] && [ "$BWP_SMTP_HOST" != "null" ]; then
+    export Mail__Smtp__Host=$BWP_SMTP_HOST
+  fi
+  
+  if [ -n "$BWP_SMTP_PORT" ] && [ "$BWP_SMTP_PORT" != "null" ]; then
+    export Mail__Smtp__Port=$BWP_SMTP_PORT
+  fi
+  
+  if [ -n "$BWP_SMTP_STARTTLS" ] && [ "$BWP_SMTP_STARTTLS" != "null" ]; then
+    export Mail__Smtp__StartTls=$BWP_SMTP_STARTTLS
+  fi
+  
+  if [ -n "$BWP_SMTP_SSL" ] && [ "$BWP_SMTP_SSL" != "null" ]; then
+    export Mail__Smtp__Ssl=$BWP_SMTP_SSL
+  fi
+  
+  if [ -n "$BWP_SMTP_SSLOVERRIDE" ] && [ "$BWP_SMTP_SSLOVERRIDE" != "null" ]; then
+    export Mail__Smtp__SslOverride=$BWP_SMTP_SSLOVERRIDE
+  fi
+  
+  if [ -n "$BWP_SMTP_TRUSTSERVER" ] && [ "$BWP_SMTP_TRUSTSERVER" != "null" ]; then
+    export Mail__Smtp__TrustServer=$BWP_SMTP_TRUSTSERVER
+  fi
+fi
+
+#########################
+# Internal & Public Url #
+#########################
+export Passwordless__ApiUrl=$PasswordlessManagement_ApiUrl
+export PasswordlessManagement__ApiUrl=$PasswordlessManagement_ApiUrl
+
+if [ "$BWP_DOMAIN" = "localhost" ] && [ "$BWP_ENABLE_SSL" != "false" ]; then
+  echo "[Configuration] ERROR: Set environment variable 'BWP_ENABLE_SSL' to 'true' when 'BWP_DOMAIN' is not 'localhost'.";
+  echo "[Configuration] ERROR: WebAuthn requires SSL when not running on 'localhost'.";
+  exit 1;
+fi
+
+if [ "$BWP_ENABLE_SSL" = "true" ]; then
+  echo "[Configuration] SSL: Enabled";
+  scheme="https"
+else
+  echo "[Configuration] SSL: Disabled";
+  scheme="http"
+fi
+if [ "$BWP_PORT" == "null" ]; then
+  echo "WARNING: 'BWP_PORT' not set, defaulting to 5701.";
+  exit 1; 
+fi
+
+export PasswordlessManagement__PublicApiUrl="$scheme://${BWP_DOMAIN:-localhost}:${BWP_PORT:-5701}/api";
+echo "[Configuration] API private: $PasswordlessManagement__ApiUrl";
+echo "[Configuration] API public: $PasswordlessManagement__PublicApiUrl";
```

Comment:
I'm curious on what the point in having the, what I think is only the internal PORT, customizable. This doesn't change the port exposed by the container, that it done in the docker run command ex. `-p 1200:5701`. 

Probably more of a question for @kspearrin as I see this is also possible in Bitwarden unified.